guest/guest.go

```diff
@@ -52,11 +58,22 @@ func (d *Default) Apply(cluster *v1.Cluster) error {
 		return fmt.Errorf("failed to found master")
 	}
 	clusterRootfs := common.DefaultTheClusterRootfsDir(cluster.Name)
+	imageCMDLayers := make([]v1.Layer, 0)
 	for i := range image.Spec.Layers {
-		if image.Spec.Layers[i].Type != common.CMDCOMMAND {
-			continue
+		if image.Spec.Layers[i].Type == common.CMDCOMMAND {
+			imageCMDLayers = append(imageCMDLayers, image.Spec.Layers[i])
 		}
-		if err := ssh.CmdAsync(masters[0], fmt.Sprintf(common.CdAndExecCmd, clusterRootfs, image.Spec.Layers[i].Value)); err != nil {
+	}
+	for i := range imageCMDLayers[0:2] {
```

Comment:
Why you split [0:2] and [2:]?

Reply:
[0:2] is "kubectl apply -f etc/tigera-operator.yaml" and "kubectl apply -f etc/custom-resources.yaml"